tests/console/check_system_info.pm

```diff
@@ -81,6 +81,15 @@ sub check_buildid {
         die "System does not upgrade to expected build ID: $build_id" if ($build eq '');
     }
 }
+# SLE-21916: change bzr to breezy
```

Comment:
I'm not sure whether this can be put in [check_system_info.pm], please double check it.
Besides, this should add conditions for SLES15SP4 only I think.